Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -10443,8 +10445,15 @@ void WebPageProxy::requestAttachmentIcon(const String& identifier, const String&
     }
 #endif
 
+    String exampleFileName;
+    size_t dotIndex = fileName.reverseFind('.');
+    if (dotIndex != notFound) {
+        String extension = fileName.substring(dotIndex);
+        exampleFileName = makeString("example", extension);
+    }
+
 #if PLATFORM(COCOA)
-    if (auto icon = iconForAttachment(fileName, contentType, title, size))
+    if (auto icon = iconForAttachment(exampleFileName, contentType, title, size))
```

Comment:
I think this should be `exampleFilename`. AFAIK, `filename` is one word, not two.